### ISSUE TITLE
Windowed mode resize issue

### ISSUE THREAD

**Issue by teelolws** [teelolws/EditModeExpanded#25]
@cdeconinck since you wrote the UI Resize scaling code, I'm hoping you can help with this issue https://github.com/teelolws/EditModeExpanded/pull/22

If a frame is placed on the right hand side, like this:
![image](https://user-images.githubusercontent.com/49792789/201496444-b1465d48-07a6-46a0-9ac5-6e1fabecbe88.png)

Then the window is resized smaller, it scrolls off the edge of the window
![image](https://user-images.githubusercontent.com/49792789/201496458-3fec5ccc-00ea-4c89-9e0e-8c78b419a563.png)

Eventually being hidden
![image](https://user-images.githubusercontent.com/49792789/201496469-d9b503d0-c6d5-4dd1-9303-1b4f063901c2.png)

I'm thinking something like... when a window resize event is detected (is there one?), check if the right side of the frame has gone out of bounds and adjust the bottom left anchor?

**Comment by teelolws** [comment]
Alternatively, instead of anchoring everything to bottom left, I detect the closest corner and anchor to that one instead

**Comment by cdeconinck** [comment]
My commit was just to fix the scaling for the EditModeManagerExpanded frame but sure i can look into that tomorrow. I've also noticed several issues with the slider used to change the scaling of each frame.
For example,  when i change the scaling of the durability frame and i press the 'revert to inital position' button, the anchor and scaling of the frame are misaligned or with the wrong size

**Comment by teelolws** [comment]
The "reset" / "revert" buttons return them to the size and position they were before my addon loaded them in. There is an issue where I don't update the options in that dialog box, though. I need to find a safe way to hook the slider, if its still open.

But to the original issue, I did find a "display size changed" event, but it doesn't tell me how much it was changed, or what direction, or... anything. I think I'm going to have to go with changing what corner I anchor everything to.
